R/01-estimateMap.R

```diff
@@ -1966,6 +1967,14 @@ modelSpread <- function(data, K, iter, burnin, MinMax, smoothConst, penalty,
                   (XX %*% betamc[usedsamples[x], ]) * sRe + mRe), 1, var))))))
 }
 
+updatePenalty <- function(penalty, bs) {
+  if (bs == "ds" & penalty == 1) {
+    penalty <- c(1, 0.5)
+  }
```

Comment:
@mgross I extracted your code into a separate function `updatePenalty()`. Additionally, Github copilot suggests to add following cases. Do these cases make any sense? Is it worth to add them?


```suggestion
 } else if (bs == "ds" & penalty == 2) {
    penalty <- c(1, 0.5, 0.25)
  } else if (bs == "sos" & penalty == 1) {
    penalty <- c(1, 0.5)
  } else if (bs == "sos" & penalty == 2) {
    penalty <- c(1, 0.5, 0.25)
  }
```

Reply:
Nein, das ist kompletter Quatsch :)

Reply:
Ja, war fast zu erwarten, viele Vorschläge waren bisher komplett daneben, obwohl einige wenige durchaus gut sind ... :sweat_smile: